docs/userGuide/plugins/disqus.mbdf

```diff
@@ -0,0 +1,50 @@
+### Plugin: Disqus 
+
+This plugin allows you to use [Disqus](https://disqus.com/) for your site.
+
+<box type="info">
+
+Before you can use this plugin, you need to **(1) setup a Disqus account** and **(2) register your site**. 
+
+You may sign up for a Disqus account and register your site [here](https://disqus.com/admin/create/). 
+
+Note that you **<ins>do not have to install Disqus on your site</ins>** as we have already done it for you. 
```

Comment:
```suggestion
Note that you **++do not have to install Disqus on your site++**, this plugin does it for you. 
```

underline style is available:
https://deploy-preview-1449--markbind-master.netlify.app/userguide/formattingcontents#text-styles

Reply:
Got it! Will make the change and keep this in mind for the future :-)